blueprints/addon/files/.travis.yml

```diff
@@ -53,7 +53,6 @@ jobs:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
     - stage: "Additional Tests"
-      env: EMBER_TRY_SCENARIO=ember-lts-2.16
     - env: EMBER_TRY_SCENARIO=ember-lts-2.18
```

Comment:
This now incorrectly triggers a build job without environment variables: https://travis-ci.org/buschtoens/ember-lazy-mount/builds/477951861

![image](https://user-images.githubusercontent.com/834636/50989268-2de22980-150f-11e9-8e11-b1ca75f45d11.png)

We need to remove the `-` from `- env:` for `ember-lts-2.18`. Super unintuitive and easy to mess API by Travis. 😞

Reply:
Oh, crap, sorry about that 😢

Could you open a PR against the release branch that fixes it?

Reply:
No worries, I was dumbfounded as well. Wouldn't have caught that mistake myself. 😊

Reply:
This is in fact _so hard_ to do right, that I _just_ fucked it up myself. 😂 

https://github.com/buschtoens/ember-lazy-mount/commit/c95124abfb79d7c45413e832dffbe3b0b44effb0
![image](https://user-images.githubusercontent.com/834636/50992735-4145c280-1518-11e9-9fcd-06974e07c414.png)